static/app/views/integrationPipeline/githubInstallationSelect.tsx

```diff
@@ -37,7 +49,7 @@ export function GithubInstallationSelect({installation_info}: GithubInstallation
     // redirect to the extensions endpoint with the form fields as query params
     // this is needed so we don't restart the pipeline loading from the original
     // OrganizationIntegrationSetupView route
-    const chosen_installation_id = id ?? installationID ?? -1;
+    const chosen_installation_id = id ?? installationID ?? '-1';
```

Comment:
nit, variables in JS should really be `camelCase`, and since it's only used one, you can probably just do this instead.
```js
const newParams = {
   ...Object.fromEntries(currentParams),
  chosen_installation_id:  id ?? installationID ?? '-1';,
};
```

Reply:
Also is it a valid submission if `chosen_installation_id`  is still `-1`? Would we want to prevent submitting without choosing an installation instead?